ReleaseNotes.md

```diff
@@ -1,5 +1,27 @@
 # Release Notes
 
+## 1.46.0
+
+### Renamings
+
+#### InspireThmeHandler
+- InspireThemes action renamed to MapLayerGroups
+- also handler class renamed to MapLayerGroupsHandler
+
+#### inspireTheme key
+
+Changed inpireTheme key to maplayerGroups in GetWmsServicesHandler and SaveLayerHandler.
+
+### control-base
+
+#### New `GetHierarchicalMapLayerGroups` action
+
+This new action handles hierarchical layerlist layers getting ordered by groups.
+
+#### New `LayerAndGroupOrder` action
+
+This new action handles layers and groups order savings.
```

Comment:
Releasenotes are gathered from the PR descriptions on releases so you don't need to modify this file. You should add this information in the description though.